src/ast/nodes/ArrowFunctionExpression.ts

```diff
@@ -28,7 +26,7 @@ export default class ArrowFunctionExpression extends Node {
 		options: ExecutionPathOptions
 	) {
 		path.length === 0 &&
-			this.scope.forEachReturnExpressionWhenCalled(callOptions, callback, options);
+		this.scope.forEachReturnExpressionWhenCalled(callOptions, callback, options);
```

Comment:
One of the few things I never got Webstorm to do right. I guess this will no longer be a problem if we add a "prettier" commit hook, but maybe in a different PR.